elixir/apps/domain/lib/domain/config/definitions.ex

```diff
@@ -609,6 +609,11 @@ defmodule Domain.Config.Definitions do
   ## Feature Flags
   ##############################################
 
+  @doc """
+  Boolean flag to enable all features
+  """
+  defconfig(:all_features_enabled, :boolean, default: false)
```

Comment:
This is a leftover which can be dropped?

Reply:
Ah, yeah, missed this one.  Good catch!